src/Elastic.Markdown/IO/DocumentationSet.cs

```diff
@@ -241,6 +264,9 @@ private static IReadOnlyCollection<INavigationItem> CreateNavigationLookup(INavi
 		if (item is ILeafNavigationItem<INavigationModel> leaf)
 			return [leaf];
 
+		if (item is CrossLinkNavigationItem crossLink)
```

Comment:
I think this will never be hit because `CrossLinkNavigationItem` is `ILeafNavigationItem<INavigationModel>`